client/src/lit/curriculum/scrim-inline.js

```diff
@@ -135,6 +159,53 @@ export class ScrimInline extends LitElement {
                   </button>
                 `}
           </div>
+          ${this.survey
+            ? html`<div class="survey">
+                ${this._voted
+                  ? html`<div class="voted">
+                      Thank you for your feedback! ❤️
+                    </div>`
+                  : html`
+                      We’re testing a new type of challenge and would love your
+                      feedback.
+                      <fieldset class="feedback">
+                        <label>
+                          How effective do you think the Scrim challenges are,
+                          compared to other tasks on this page?
+                        </label>
```

Comment:
From a user perspective, the different font sizes confuse me. Also, it is not necessarily immediately clear that this is a survey, so a visual indicator would help.

Reply:
Again, text content has been developed elswhere.